pages/c/_cluster/apps/install-steps.vue

```diff
@@ -0,0 +1,1131 @@
+            <Tabbed
+              v-else-if="hasQuestions && showQuestions"
+              ref="tabs"
+              :side-tabs="true"
+              :class="{'with-name': showNameEditor}"
+              class="step__values__content"
+              @changed="tabChanged($event)"
+            >
+              <Questions
+                v-model="chartValues"
+                :mode="mode"
+                :chart="chart"
+                :version="version"
+                :version-info="versionInfo"
+                :target-namespace="targetNamespace"
+              />
+            </Tabbed>
+            <!-- Values (as YAML) -->
+            <template v-else>
+              <YamlEditor
+                ref="yaml"
+                v-model="valuesYaml"
+                class="step__values__content"
+                :scrolling="true"
+                :initial-yaml-values="originalYamlValues"
+                :editor-mode="editorMode"
+                :hide-preview-buttons="true"
+              />
+            </template>
+          </div>
+        </div>
+
+        <!-- Confirm loss of changes on toggle from yaml/preview to form -->
+        <ResourceCancelModal ref="cancelModal" :is-cancel-modal="false" :is-form="true" @cancel-cancel="preShowPreview=true" @confirm-cancel="showPreview = false;"></ResourceCancelModal>
+      </template>
+      <template #helmCli>
+        <div><Checkbox v-if="existing" v-model="cleanupOnFail" :label="t('catalog.install.helm.cleanupOnFail')" /></div>
+        <div><Checkbox v-if="!existing" v-model="crds" :label="t('catalog.install.helm.crds')" /></div>
+        <div><Checkbox v-model="hooks" :label="t('catalog.install.helm.hooks')" /></div>
+        <div><Checkbox v-if="existing" v-model="force" :label="t('catalog.install.helm.force')" /></div>
+        <div><Checkbox v-if="existing" v-model="resetValues" :label="t('catalog.install.helm.resetValues')" /></div>
+        <div><Checkbox v-if="!existing" v-model="openApi" :label="t('catalog.install.helm.openapi')" /></div>
+        <div><Checkbox v-model="wait" :label="t('catalog.install.helm.wait')" /></div>
+        <div style="display: block; max-width: 400px;" class="mt-10">
+          <UnitInput
+            v-model.number="timeout"
+            :label="t('catalog.install.helm.timeout.label')"
+            :suffix="t('catalog.install.helm.timeout.unit', {value: timeout})"
+          />
+        </div>
+        <div style="display: block; max-width: 400px;" class="mt-10">
+          <UnitInput
+            v-if="existing"
+            v-model.number="historyMax"
+            :label="t('catalog.install.helm.historyMax.label')"
+            :suffix="t('catalog.install.helm.historyMax.unit', {value: historyMax})"
+          />
+        </div>
+      </template>
+    </Wizard>
+    <div class="slideIn" :class="{'hide': false, 'slideIn__show': showSlideIn}">
+      <button type="button" class="close btn btn-sm role-secondary" @click="showSlideIn = false">
```

Comment:
esc to close

Reply:
I haven't applied this. The slide in is more sticky (user edits yaml whilst out)